Documentation/Signoffs/SteeringSystem.md

```diff
@@ -8,8 +8,10 @@ the data into a local microcontroller which will interpret the data and relay it
 ### Specifications & Constraints
 - Must measure angle of rotation of the steering wheel
 - Must use the Nissan Leaf OEM steering wheel as the input
-- Angle Shall be no more than 960 degrees to the left and right (2 and 2/3 rotations)
+- Angle shall be no more than 960 degrees from the left boundary to the right boundary (2 and 2/3 rotations)
 - Error of the angle must be within 1% of the true steering angle
+- Must not lose steering position when power is cycled
+- System must be small enough to fit in the area of the vehicle without interfering with the driver or other subsytems
 
 The simulation requires an input for the steering angle of the vehicle. In the Nissan Leaf, the steering
```

Comment:
This is not correct. There is no reason the measuring must be limited to this range.